.github/workflows/release.yml

```diff
@@ -27,7 +27,9 @@ jobs:
 
       - name: Build tarball
         shell: bash
-        run: tar czf ${{ env.WORKING_DIR }}.tar.gz ${{ env.WORKING_DIR }}
+        run: |
+          rm -r ${{ env.WORKING_DIR }}/{.git,.github}
```

Comment:
Do we need to use `-rf` since it's a directory?

Reply:
@tunbola  -r is enough.